integration_tests/features/WalletFFI.feature

```diff
@@ -1,129 +1,136 @@
 @wallet-ffi
 Feature: Wallet FFI
+    # Increase heap memory available to nodejs if frequent crashing occurs with
+    # error being be similar to this: `0x1a32cd5 V8_Fatal(char const*, ...)`
 
-    Scenario: As a client I want to send Tari to a Public Key
-    # It's a subtest of "As a client I want to retrieve a list of transactions I have made and received"
-
-    Scenario: As a client I want to specify a custom fee when I send tari
-    # It's a subtest of "As a client I want to retrieve a list of transactions I have made and received"
-
-    Scenario: As a client I want to receive Tari via my Public Key while I am online
-    # It's a subtest of "As a client I want to retrieve a list of transactions I have made and received"
+    # It's just calling the encrypt function, we don't test if it's actually encrypted
```

Comment:
These tests are so different from the previous file it makes it impossible to know what's changed. Did the order change or are these entirely new tests? Are the previous tests still intact?